packages/babel-types/test/cloning.js

```diff
@@ -91,6 +92,7 @@ describe("cloneNode", function() {
         typeAnnotation: t.anyTypeAnnotation(),
       }),
     ]);
+    node.loc = {};
     const cloned = t.cloneNode(node, /* deep */ true, /* withoutLoc */ true);
     expect(cloned.loc).toBeNull();
```

Comment:
Check that `cloned.id.loc` is null, too.

Reply:
Thanks, somehow `cloned.id.loc` is always `{}` not null! any idea why?